examples/calendar/backend/services/time/controller.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { NextFunction, Request, Response } from "express";
+import { Time } from "../../models/time.model";
+import { Appointment } from "../../models/appointment.model";
+import { Op } from "sequelize";
+import {Status} from "./../../db";
+import { Logger } from "tslog";
+
+
+const log: Logger = new Logger({ name: "errorLogger" });
+// Create time availablity for the user
+
+export const create = async (req: Request, res: Response, next: NextFunction) => {
+	console.log("req", req.body);
+	const { timeStarts, timeEnds } = req.body.availableTimes;
+	if (!(req as any).hasOwnProperty("user")) {
+		return res.status(401).send({ error: "No User found!" });
+	}
+	const userId = (req as any).user.payload.id;
```

Comment:
payload may not have an id here

Reply:
Added checks

Reply:
Can you please check that the code was pushed?

Reply:
Yes, check line 15 in the same file. (https://github.com/eea-oasis/baseline/blob/c759b8dfba3e810be8a6a611849d1d6e204949f0/examples/calendar/backend/services/time/controller.ts#L15)